test/onevariable.jl

```diff
@@ -263,6 +263,17 @@ using Base.Test
     @test string(ta(0)^3-3) == " - 3 + 1 t³ + 𝒪(t¹⁶)"
     @test TaylorSeries.pretty_print(ta(3im)) == " ( 3 im )  + ( 1 ) t + 𝒪(t¹⁶)"
     @test string(Taylor1([1,2,3,4,5], 2)) == string(Taylor1([1,2,3]))
+
```

Comment:
My suggestion here it to test the `norm` of something like `t_C = complex(3.0,4.0) * t_a` against the `norm` of `complex(3.0,4.0) * a`; as defined below, `t_C` is only a constant Taylor series.

Reply:
I'll change this too!